### PR TITLE
Bug 1717623 - fix CSV for OLM and ART

### PR DESCRIPTION
**Pull request by jcantrill**
This PR fixes up the CSV for z stream release to adhere to ART and OLM expectations

### CODE REVIEW

File: manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ metadata:
     containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
+    olm.skipRange: ">=4.1.0 <4.2"
```
Comment (chancez) [inline manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]:
```suggestion
    olm.skipRange: ">=4.1.0 <4.1.1"
```

File: manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -68,6 +69,10 @@ spec:
 
   keywords: ['elasticsearch', 'jaeger']
 
+  replaces: elasticsearch-operator.v4.1.0
```
Comment (chancez) [inline manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: I don't believe this is necessary for z streams, only for something like 4.1 -> 4.2

Comment (chancez) [reply manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: @ecordell @jwforres can you confirm?

File: manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -68,6 +69,10 @@ spec:
...
+  # this will eventually be populated for any subsequent z-stream fixes
+  skips: []
```
Comment (ecordell) [inline manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: This isn't needed for z-streams, since the `skipRange` will take priority

Comment (jcantrill) [reply manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: @ecordell so I leave `replaces` but remove the empty `skip`?

Comment (jcantrill) [reply manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: removed both items

**Comment by bparees** [comment]
looks right as i understood it, but will hold for @ecordell and @jwforres 

also @jcantrill i think you should remove this line while you're in here:
https://github.com/openshift/elasticsearch-operator/blob/master/manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml#L36

so the "example" just guides people to use the default image.

**Comment by jcantrill** [comment]
> also @jcantrill i think you should remove this line while you're in here:
> https://github.com/openshift/elasticsearch-operator/blob/master/manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml#L36

done

**Comment by jcantrill** [comment]
> Also need to update package.yml to match `metadata.name` still

modified.

**Comment by jwforres** [comment]
/lgtm
but im not sure if @jcantrill is done and ready to merge so i'll add a
/hold
and logging team can cancel when ready to merge

**Comment by bparees** [comment]
@jcantrill please create the CLO version of this PR asap, thanks

**Pull request merged**